docs/source/specs/openapi.json

```diff
@@ -93,6 +93,38 @@
                 "true"
               ]
             }
+          },
+          {
+            "in": "query",
+            "name": "status",
+            "required": false,
+            "description": "Parameter for filtering resource based on status.",
+            "schema": {
+              "type": "string",
```

Comment:
Can you allow filtering by multiple statuses? This is the way the UI is currently setup:
![image](https://user-images.githubusercontent.com/47335686/114876112-ffc5b100-9dcb-11eb-8d90-89e5d4586b13.png)

this calls: `/api/rbac/v1/cross-account-requests/?query_by=user_id&offset=0&limit=10&status=approved%2Cdenied%2Ccancelled&sort_by=created&sort_direction=desc` which I'd expect to return approved, denied, and cancelled requests.

Reply:
Sure, will add this